src/api/api.ts

```diff
@@ -95,3 +95,25 @@ export const Login = async (params: ILogin): Promise<IResponse> => {
 export const getUserInfo = (params: IUser): Promise<IResponse> => {
   return axiosInstance.post('user/getInfo', params).then((res) => res.data);
 };
+
+export const getApplications = async (): Promise<IResponse> => {
+  return await axios.get('/mock/application.json', {
```

Comment:
It is recommended to encapsulate a unified API request class for unified error handling.